graders/java/Dockerfile

```
@@ -4,20 +4,19 @@ LABEL maintainer="jonatan@yorku.ca"
 
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update && apt-get install -y make openjdk-21-jdk-headless jq && apt-get clean
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends make openjdk-21-jdk-headless jq && apt-get clean && \
```

Nit, split to multiple lines for consistency?

```suggestion
    apt-get install -y --no-install-recommends make openjdk-21-jdk-headless jq && \
    apt-get clean && \
```